Zend/zend_API.c

```diff
@@ -45,6 +45,17 @@ static zend_module_entry **module_post_deactivate_handlers;
 
 static zend_class_entry  **class_cleanup_handlers;
 
+ZEND_API zval * zend_get_array_value(zend_string *key, zval * argument_array)
+{
+    zval * retval;
+    HashTable * argument_array_table;
+    argument_array_table = Z_ARRVAL_P(argument_array);
+    if ((retval = zend_hash_find(argument_array_table, key)) != NULL) {
```

Comment:
If you're returning `NULL` anyway on failure, then you may as well just remove this branch and directly return the result of `zend_hash_find`